src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -32,13 +33,19 @@ internal TagHelperOutput(string tagName)
         /// </summary>
         /// <param name="tagName">The HTML element's tag name.</param>
         /// <param name="attributes">The HTML attributes.</param>
-        public TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> attributes)
+        /// <param name="htmlEncoder">The <see cref="IHtmlEncoder"/> to be used
```

Comment:
teensy nit: don't need "to be" (not one of my better suggestions if that was mine)